sql/mysql_db_structure.sql

```diff
@@ -319,12 +324,13 @@ CREATE TABLE `judging_run` (
 
 CREATE TABLE `judging_run_output` (
   `runid` int(4) unsigned NOT NULL COMMENT 'Run ID',
-  `output_run` longblob DEFAULT NULL COMMENT 'Output of running the program',
-  `output_diff` longblob DEFAULT NULL COMMENT 'Diffing the program output and testcase output',
-  `output_error` longblob DEFAULT NULL COMMENT 'Standard error output of the program',
-  `output_system` longblob DEFAULT NULL COMMENT 'Judging system output',
+  `output_run` longblob DEFAULT NULL COMMENT 'Output of running the program(DC2Type:blobtext)',
```

Comment:
I have created `blobtext` for fields that are blobs in the DB but should not be a `resource` in PHP.